test/testE2E.t.sol

```diff
@@ -13,12 +13,15 @@ import "../src/ParachainRegistry.sol";
 import "../src/Parachain.sol";
 import "../src/ParachainStaking.sol";
 import "../src/ParachainGovernance.sol";
+import "./testParachain.t.sol";
 
 contract E2ETests is Test {
     TestToken public token;
     ParachainRegistry public registry;
     ParachainStaking public staking;
     ParachainGovernance public gov;
+    ParachainTest public parachainTest;
+    Parachain public parachainContract;
```

Comment:
I assume that ParachainTest is the one that you want, extending Parachain already, so can remove line 24 and just use the parachainTest field?